resources/definitions/ideagen3D_sapphire_plus.def.json

```
@@ -1,5 +1,5 @@
 {
-    "version": 2,
+    "version": 3,
```

This is a version number of the file format, and should be left at 2 since it's the most recent version.

```suggestion
    "version": 2,
```